lib/faraday/adapter/em_synchrony.rb

```diff
@@ -145,6 +145,8 @@ def call_block(block)
     warn 'Warning: no such file to load -- openssl. ' \
       'Make sure it is installed if you want HTTPS support'
   else
-    require 'faraday/adapter/em_http_ssl_patch'
+    if EventMachine::HttpRequest::VERSION < '1.1.6'
```

Comment:
And here require version as well. They are independent adapters.

Reply:
Sorry about that - just pushed this change